framework/data/Sort.php

```diff
@@ -134,6 +135,12 @@ class Sort extends Object
      */
     public $sortParam = 'sort';
     /**
+     * @var string the name of the parameter that specifies the page number in
+     * the url. This will be used to unset said value on the sort link.
+     * @since 2.0.11
+     */
+    public $pageParam = 'page';
```

Comment:
could be `null` by default, and if set it will be used. this way we can introduce it backwards compatible.